README.md

```diff
@@ -1,239 +1,158 @@
-
-You can customize the clock skew as shown below:     
+Finally, complete the authentication and obtain the tokens by calling `handle()` on the `AuthenticationController`.
 
 ```java
-AuthenticationController authController = AuthenticationController.newBuilder("domain", "clientId", "clientSecret")
-    .withClockSkew(60 * 2)   //2 minutes
-    .build();
-```
-
-#### HTTP Logging 
-Once you have created the instance of the `AuthenticationController` you can enable HTTP logging for all Requests and Responses to debug a specific endpoint. **This will log everything including sensitive information** so don't use it in a production environment.
-
-```java
-authController.setLoggingEnabled(true);
+@WebServlet(urlPatterns = {"/callback"})
+public class CallbackServlet extends HttpServlet {
+    
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse res) throws IOException, ServletException {
+        try {
+            // authentication complete; the tokens can be stored as needed
+            Tokens tokens = AuthenticationControllerProvider
+                    .getInstance()
+                    .handle(req, res);
+            res.sendRedirect("URL-AFTER-AUTHENTICATED");
+        } catch (IdentityVerificationException e) {
+            // handle authentication error
+        }
+    }
+}
 ```
 
-## Issue Reporting
-
-If you have found a bug or if you have a feature request, please report them at this repository issues section. Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
-
-## What is Auth0?
-
-Auth0 helps you to:
-
-* Add authentication with [multiple authentication sources](https://docs.auth0.com/identityproviders), either social like **Google, Facebook, Microsoft Account, LinkedIn, GitHub, Twitter, Box, Salesforce, among others**, or enterprise identity systems like **Windows Azure AD, Google Apps, Active Directory, ADFS or any SAML Identity Provider**.
-* Add authentication through more traditional **[username/password databases](https://docs.auth0.com/mysql-connection-tutorial)**.
-* Add support for **[linking different user accounts](https://docs.auth0.com/link-accounts)** with the same user.
-* Support for generating signed [Json Web Tokens](https://docs.auth0.com/jwt) to call your APIs and **flow the user identity** securely.
-* Analytics of how, when and where users are logging in.
-* Pull data from other sources and add it to the user profile, through [JavaScript rules](https://docs.auth0.com/rules).
+That's it! You have authenticated the user using Auth0.
 
-## Create a free account in Auth0
+## API Reference
 
-1. Go to [Auth0](https://auth0.com) and click Sign Up.
-2. Use Google, GitHub or Microsoft Account to login.
+- [JavaDocs](https://javadoc.io/doc/com.auth0/auth0-java-mvc-common/latest)
```

Comment:
Same here, this 404s for me. Or are we updating the package name in future?

Reply:
Just a bad copy/paste, updated in newer commit